packages/chain-adapters/src/types.ts

```diff
@@ -131,6 +131,7 @@ type ChainSignTx = {
 export type SignTx<T extends ChainId> = T extends keyof ChainSignTx ? ChainSignTx[T] : never
 
 export type BuildSendTxInput<T extends ChainId> = {
+  from?: string
```

Comment:
only other thought would be to move from to the `chainSpecific` section since it isn't relevant for evm/cosmossdk chains

Reply:
Ab-solutely, much better umbrella - interestingly `getFeeData` for EVM chains also takes a `chainSpecific.from` due to the way ERC20s send maxes gas estimation work, so that will keep things consistent!

99ce9421753d1fb2d88deced3dceea09c0a62d2e